.pre-commit-config.yaml

```diff
@@ -21,9 +21,88 @@ repos:
     hooks:
       - id: ruff
         args: ["--fix"]
-      # - id: ruff-format
+      #- id: ruff-format
   - repo: https://github.com/PyCQA/bandit
```

Comment:
Bandit could be replaced by the `S` rules:  https://docs.astral.sh/ruff/rules/#flake8-bandit-s

Reply:
Done, let me know how it looks